src/Microsoft.ML.Data/Transforms/Hashing.cs

```diff
@@ -834,6 +1083,66 @@ private void AddMetaKeyValues(int i, DataViewSchema.Annotations.Builder builder)
             }
 
             protected override Delegate MakeGetter(DataViewRow input, int iinfo, Func<int, bool> activeOutput, out Action disposer) => _parent.GetGetterCore(input, iinfo, out disposer);
+
+            private bool SaveAsOnnxCore(OnnxContext ctx, int iinfo, string srcVariable, string dstVariable)
```

Comment:
>SaveAsOnnxCore [](start = 25, length = 14)

Should `NumberOfBits` and `UseOrderedHashing` influence the output here?

Reply:
yes, mask is dependant on NumberofBits, and we aslo need UseOrderedHashing, but we can get that info from _parent._columns[iinfo]

Reply:
But I don't see this information being used anywhere in `SaveOnnxCore`.

---
In reply to: [391310537](https://github.com/dotnet/machinelearning/pull/4596#discussion_r391310537) [](ancestors = 391310537)